README.md

```diff
@@ -1,6 +1,7 @@
 # prebuild-install
 
 [![Build Status](https://travis-ci.org/prebuild/prebuild-install.svg?branch=master)](https://travis-ci.org/prebuild/prebuild-install)
+[![Build status](https://ci.appveyor.com/api/projects/status/6v6hxxwgjrr99pc8/branch/master?svg=true)](https://ci.appveyor.com/project/mathiask88/prebuild-install)
```

Comment:
Possible to get this under `prebuild/prebuild-install`? No biggie though.

Reply:
Is this possible with my appveyor account? I get `Project not found or access denied.` Do we have a `prebuild` appveyor account?

Reply:
Nevermind, lets merge this now and tweak stuff later.

Reply:
We could take the same approach as in Level: a github machine user (`prebuild-ci`) as main AppVeyor account, and subaccounts for maintainers.